web/src/views/SubmissionPortal/Components/DataTypes.vue

```diff
@@ -10,14 +10,52 @@ export default defineComponent({
       type: String,
       default: 'Data Types',
     },
+    showDataCompatibilityQuestions: {
+      type: Boolean,
+      default: true,
+    },
   },
-  setup() {
+  setup(props, { emit }) {
```

Comment:
I don't think you need to pass props here because it doesn't look like you are accessing them in the setup function.

Reply:
Oh, I was just sort of blindly following a pattern I saw elsewhere. Maybe this is a dumb Vue question then, but is it possible to just get `emit` without getting `props`?

Reply:
Got it, thanks.